dispatch/migrations/0021_topic_slug_unique.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
```

Comment:
Was it necessary to create a whole new migration file?

Reply:
When applying the reverse_set_defaults function, I ran into a duplicate key error. So the second migration file is just to ensure that before reverting migration 0020, the fields do not have to be unique. Of course as I write this I realize I set the slugs to None, so you are probably right, this 0021 isn't necessary.